libs/sdk-ui-pivot/src/CorePivotTable.tsx

```diff
@@ -352,6 +357,23 @@ export class CorePivotTableAgImpl extends React.Component<ICorePivotTableProps,
         );
     }
 
+    /**
+     * Checks DOM if height has been set on PivotTable parent wrapper. If height has not been set,
+     * PivotTable will not be rendered correctly. Therefore, we need to inform the user at least on the console
+     * on what is happening.
+     *
+     * @private
+     */
+    private alertParentWrapperMissingHeight(): void {
+        const parentWrapper = document.getElementById(this.pivotTableId)?.parentElement;
+        const parentHeight = parentWrapper?.style?.height || "0";
+        if (isEmpty(parentHeight) || parseInt(parentHeight) < 20) {
+            console.warn(
+                `The wrapper height of the pivot table has not been set or is suspiciously small. This might cause pivot table rendering issues. If so, please set an appropriate height for the wrapper. Use "${this.pivotTableId}" to find the PivotTable element in the DOM, which will help you to identify its wrapper.`,
```

Comment:
Please provide an example of how to do it (document.getElementById for example).